src/nunit.analyzers/Extensions/AttributeArgumentTypedConstantExtensions.cs

```diff
@@ -210,6 +210,10 @@ private static bool HasBuiltInImplicitConversion(ITypeSymbol argumentType, IType
                     }
                 }
             }
+            else if (targetTypeSymbol.GetAttributes().Any(att => att.AttributeClass?.GetFullMetadataName() == typeof(TypeConverterAttribute).FullName))
```

Comment:
I'm not behind a computer, but isn't TypeConverter not already dealt with?
You added a vase when targetType is null?
The CustomTypeConverter in your test doesn't override the Can convert from tested in the existing code.

Reply:
TypeConverters are only dealt with for strings. And the code tries to check for types it can access the code from. In most cases that is not possible. So basically, you can not know if it is supported or not, but because of that, you should report on it. Specially because the default severity is an error, and it runs perfectly.

Reply:
Thanks. I am on leave without access to a development PC until middle of July. I will look further then.

Reply:
Sorry, I forgot to look at.
Your added case can be removed. Instead we can remove the condition on the line below.